lldb/scripts/copy-static-bindings.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+"""SWIG Static Binding Copier
+
+This script copies over the Python bindings generated by SWIG from the build
+directory to the source directory. Before using the script, make sure you have
+LLDB_USE_STATIC_BINDINGS set to OFF by looking at CMakeCache.txt in the LLDB
```

Comment:
Nit: I guess we could check that and exit early with an error message to change CMake option ?

Reply:
I considered doing that, but there are many values that CMake accepts, such as `On`, `on`, `1`, etc. It didn't seem worth maintaining a shadow implementation for that.

Reply:
Fair 💯